src/modules/endpoint/DatabaseConnector.js

```diff
@@ -171,19 +185,55 @@ class DatabaseConnector {
       throw new Error('Language is not set in DatabaseContext!')
     }
 
-    const path = this.getMetaCitiesPath()
-    const fileExists: boolean = await RNFetchblob.fs.exists(path)
-
-    if (!fileExists) {
+    const metaData = await this.loadMetaCities()
+    if (!metaData) {
       return null
     }
 
-    const currentCityMetaData: MetaCitiesJsonType = JSON.parse(await this.readFile(path))
     // eslint-disable-next-line camelcase
-    const lastUpdate = currentCityMetaData[cityCode]?.languages[languageCode]?.last_update
+    const lastUpdate = metaData[cityCode]?.languages[languageCode]?.last_update
     return lastUpdate ? moment(lastUpdate, moment.ISO_8601) : null
   }
 
+  async loadLastUsages (): Promise<Array<CityLastUsageType>> {
+    const metaData = await this.loadMetaCities()
+    if (!metaData) {
+      return []
+    }
+
+    return map<MetaCitiesEntryJsonType, MetaCitiesJsonType, CityLastUsageType>(
+      metaData, (value, key) => ({
+        city: key,
+        lastUsage: value.last_usage ? moment(value.last_usage, moment.ISO_8601) : null
+      })
+    )
+  }
+
+  async updateLastUsages (lastUsages: Array<CityLastUsageType>) {
```

Comment:
Is it ever necessary to update more than one `lastUsage`? If no, we should probably only define `updateLastUsage(context)` (the one below). Otherwise it's more complicated to test.
Edit: I saw, that you used this. In consideration with my other comments, you might not need it though...

Reply:
Normally it's should not be needed since old resource caches should be deleted on every `storeResourceCache`, so there should never be more than one to delete. If a resource cache appears suddenly from somewhere though, we may need it :P

Reply:
I was looking from the perspective, that `lastUsage` is never updated to `null`, since it made no sense to me. Then you would only ever update a lastUsage for a city once you use the city.

Reply:
Ah yeah :+1: